earthpy/tests/test_spatial.py

```diff
@@ -1,6 +1,39 @@
-from earthpy import spatial
+import pandas as pd
+import pytest
+from shapely.geometry import Polygon, Point
+import geopandas as gpd
+import earthpy.spatial as es
 
 
-def test_dummy():
-    """"Dummy test that will pass."""
-    assert 1 == 1
+def test_extent_to_json():
+    """"Unit tests for extent_to_json()."""
+    # Giving a list [minx, miny, maxx, maxy] makes a polygon
+    list_out = es.extent_to_json([0, 0, 1, 1])
+    assert list_out['type'] == 'Polygon'
+
+    # The polygon is the unit square
+    list_poly = Polygon(list_out['coordinates'][0])
+    assert list_poly.area == 1
+    assert list_poly.length == 4
+
+    # Providing a GeoDataFrame creates identical output
```

Comment:
Should we make a geodataframe builder function @mbjoseph that accepts a list of points and created a gdf? i can create an issue. i had this thought last night.

Reply:
Sure! This seems like a useful utility to have in our test suite. I can think of at least a few places where it would be useful:

- here, to make a GeoDataFrame that has some extent
- to test the `crop` function edge cases
- if we ever do anything with rasterizing points or drawing polygons around points